trueos-handbook/introducing.rst

```diff
@@ -718,55 +770,104 @@ A virtualized environment allows a user to test drive an operating
+
+.. TODO: Add additional information for using bhyve
+
+.. index:: virtualbox
+.. _VirtualBox:
+
+VirtualBox
+----------
+
+VirtualBox is a popular virtualization software available in |trueos|.
+Installing VirtualBox through the |sysadm| :sysclbk:`AppCafe <appcafe>`
+GUI, or :command:`pkg install virtualbox-ose`  on the command line, will
+install all required dependencies. If installing |trueos| inside a
+virtual machine, or "guest" installing the virtualbox-ose-additions
+package (also known as VirtualBox Guest Additions) will greatly improve
+the performance of |trueos| or any other guest operating system. The
+guest additions add mouse pointer integration, shared folders between
+the host and guest (depending on the guest), better video support, and
+a shared clipboard.
+  .. note:: VirtualBox does not currently support the shared folders
+	    feature with a |trueos| guest. To share files between the
+	    host, and a |trueos| guest, use NFS share.
+
+Please see the `VirtualBox <https://www.virtualbox.org/>`_ website for
+additional information.
+`VirtualBox Guest Additions <http://www.virtualbox.org/manual/ch04.html>`_
+for more information about what is supported and how to use them.
+
+
 The guest additions add mouse pointer integration, shared folders
 between the host and guest, better video support, and a shared
 clipboard.
 
 .. note:: The first time running VirtualBox on a |trueos| system, a
-   background script automatically gives the user account the
-   permissions required to run this application. This might break any
-   existing shortcuts to VirtualBox. To fix the shortcut, logout and in
-   again.
-
-If the computer is running another operating system, download the binary
-for the specific operating system from the
-`VirtualBox Downloads page <https://www.virtualbox.org/wiki/Downloads>`_.
-VirtualBox runs on Windows, Linux, Macintosh, and OpenSolaris. It
-supports a large number of virtual machine installable operating
-systems.
-
-This section describes how to prepare VirtualBox for an installation of
-|trueos| using an :file:`.iso` file.
-
-.. index:: create virtual machine for an iso
-.. _Creating a Virtual Machine for an ISO File:
-
-Creating a Virtual Machine for an ISO File
-------------------------------------------
-
-Once the |trueos| ISO is downloaded and VirtualBox installed on the
-current system, create a virtual machine and use the ISO to install
-|trueos| into the virtual machine. The virtual machine must meet several
-minimum requirements and this section will demonstrate how to configure
-these:
-
-* 1024 MB base memory size.
-
-* A virtual disk **at least 20 GB in size** for a server installation or
-  **at least 50 GB in size** for a desktop installation.
+   background script automatically gives the user account that started
+   VirtualBox the permissions required to run the application. This
+   might break existing shortcuts to VirtualBox. To fix the shortcut,
+   logout and in again.
```

Comment:
"logout" -> "log out".

Technical note:
Before a user tries to start virtualbox for the first time, they need to be a part of the "vboxusers" group in order to prevent the permissions issues later on.
1. Install package
2. Add current user to "vboxusers" group (CLI or SysAdm User Manager).
3. Log out and log in to activate new group settings.
4. Now the VirtualBox utility can be launched normally.

Reply:
I recently installed VirtualBox. Instructions for loading the vboxdrv kernel module, adding all users that want to run VirtualBox to the vboxusers group, and to reboot after performing these steps are provided in the post-install output. I actually have a copy of these instructions handy. Is this something that can/should be included here? Maybe pared down to just the commands?